middleware/error_handler.go

```diff
@@ -38,7 +40,7 @@ func ErrorHandler(service *goa.Service, verbose bool) goa.Middleware {
 					reqID = shortID()
 					ctx = context.WithValue(ctx, reqIDKey, reqID)
 				}
-				goa.LogError(ctx, "uncaught error", "id", reqID, "msg", respBody)
+				goa.LogError(ctx, fmt.Sprintf("uncaught error : %+v", e), "id", reqID, "msg", respBody)
```

Comment:
Is there a reason for not using a key/value pair to log the error? something like `goa.LogError(ctx, "uncaught error", "err", e, "id", reqID, "msg", respBody)`